src/main/groovy/com/bmuschko/gradle/docker/utils/IOUtils.groovy

```diff
@@ -44,6 +44,5 @@ final class IOUtils {
         ProgressLoggerFactory factory = registry.get(ProgressLoggerFactory)
         ProgressLogger progressLogger = factory.newOperation(Objects.requireNonNull(clazz))
         progressLogger.setDescription("ProgressLogger for ${clazz.getSimpleName()}")
-        progressLogger.setLoggingHeader(null)
```

Comment:
Will this change make it completely Gradle 6.0-compatible? The creation of a ProgressLogger is still OK?

Reply:
Yes, only some methods have been removed. See [this commit](https://github.com/gradle/gradle/commit/4ca2ff6c4e8f1bb406dcc9a051899256a2e3586e).

Reply:
So technically it's still there just the setter is removed. In theory we should be able to do:
```
progressLogger.loggingHeader = null
```
Though that seems a redundant operation. Just putting it out there should folks stumble upon this in the future.